CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # UNRELEASED
 
+**BREAKING CHANGES**
+
+-   Update matrix-rust-sdk to `9b05d0d82`, which includes:
+
+    -   `Device.requestVerification`, `UserIdentities.requestVerification`, and
+        `UserIdentities.verificationRequestContent` is not async anymore.
+        ([#3513](https://github.com/matrix-org/matrix-rust-sdk/pull/3513))
+
+    -   Use the server name in the `QrCodeData` instead of the homeserver URL.
+        ([#3537](https://github.com/matrix-org/matrix-rust-sdk/pull/3537))
```

Comment:
Not sure if this is a breaking change either?

Reply:
Well we did rename the field, and the type of the field, thus the getter in the bindings was also renamed.

Reply:
ah, it might be good to call out that rename then?

Reply:
https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/125